07.bowling_object/game.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative './shot'
+require_relative './frame'
+
+class Game
+  def initialize(marks)
+    @marks = marks.split(',')
+  end
+
+  def build_frames
+    frames = []
+    10.times { frames << [] }
+    frames_index = 0
+    @marks.each do |mark|
+      if frames_index == 9
+        frames[frames_index] << (mark == 'X' ? 'X' : mark)
```

Comment:
これは
```suggestion
        frames[frames_index] << mark
```
と同じではないでしょうか？